lib/ui/text.dart

```diff
@@ -423,12 +439,14 @@ class ParagraphStyle {
     TextAlign textAlign,
     FontWeight fontWeight,
     FontStyle fontStyle,
+    TextLineBreakMode textLineBreakMode,
     String fontFamily,
     double fontSize,
     double lineHeight
```

Comment:
Eventually, we'll want to add a "lineCount" field here that says how many lines of text to show.  We don't need to add that in this patch (obviously), but that's what's guiding some of these naming choices.